packages/react-app/src/components/HardhatEVM.jsx

```diff
@@ -0,0 +1,175 @@
+import { SendOutlined } from "@ant-design/icons";
+import { Button, Input, Tooltip, Drawer, Form, Row, Col, Select, DatePicker, Divider } from "antd";
+import { useLookupAddress } from "eth-hooks";
+import React, { useCallback, useEffect, useState } from "react";
+import Blockies from "react-blockies";
+import { Transactor } from "../helpers";
+import Wallet from "./Wallet";
+import {
+  useOnBlock,
+} from "../hooks";
+import { formatEther, parseEther } from "@ethersproject/units";
+import { parse } from "graphql";
+
+const { Option } = Select;
+const { utils, BigNumber } = require("ethers");
+
+/*
```

Comment:
for new components, i would suggest we use JS doc format.  will help future documentation, and hints for new developers.

below is an example from a hook
```
/**
 * Checks whether a contract exists on the blockchain, returns true if it exists, otherwise false
 * 
  ~ Features ~
  - Provide contractAddress to check if the contract is deployed
  - Change provider to check contract address on different chains (ex. mainnetProvider)
 * @param provider 
 * @param contractAddress 
 * @returns 
 */
```